src/qt/walletmodel.cpp

```diff
@@ -414,12 +390,7 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
 
         CReserveKey *keyChange = transaction.getPossibleKeyChange();
         CValidationState state;
-        // the new IX system does not require explicit IX messages
-        std::string strCommand = NetMsgType::TX;
-        if (recipients[0].fUseInstantSend && IsOldInstantSendEnabled()) {
-            strCommand = NetMsgType::TXLOCKREQUEST;
-        }
-        if(!wallet->CommitTransaction(*newTx, *keyChange, g_connman.get(), state, strCommand))
+        if(!wallet->CommitTransaction(*newTx, *keyChange, g_connman.get(), state))
```

Comment:
`if(` -> `if (`

Reply:
Out of scope of this PR...wallet.cpp is generally loaded with style infringements, but fixing them would lead to many merge conflicts in backports.

Reply:
Ah sorry, this is walletmodel.cpp and not wallet.cpp